dbmigration/20220602182139_add_prow_pull_requests.sql

```diff
@@ -0,0 +1,60 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE public.prow_pull_requests (
```

Comment:
This migration was challenging to create, I ended up using AutoMigrate from gorm to generate the schema.